<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debugTool</class>
 <widget class="QMainWindow" name="debugTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DebugTool</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(50, 50, 50);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnTracebackTest">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(221, 221, 221);</string>
       </property>
       <property name="text">
        <string>報錯測試</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenLogPath">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(221, 221, 221);</string>
       </property>
       <property name="text">
        <string>開啟errorLog</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoggerDebugMode">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(221, 221, 221);</string>
       </property>
       <property name="text">
        <string>切換Log層級(目前：Error)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenConsole">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(221, 221, 221);</string>
       </property>
       <property name="text">
        <string>打開Console</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDumpAllPage">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(221, 221, 221);</string>
       </property>
       <property name="text">
        <string>dumpAllPage(31001)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDumpScoreData">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(221, 221, 221);</string>
       </property>
       <property name="text">
        <string>dumpScoreData</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>210</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
